src/classes/STG_InstallScript.cls

```diff
@@ -206,11 +201,10 @@ global without sharing class STG_InstallScript implements InstallHandler {
         }
     }
 
-     /*******************************************************************************************************
+    /*******************************************************************************************************
     * @description Disables Preferred Phone handling when upgrading to 1.82
     * @return void
     */
-    
     global static void disablePreferredPhoneEnforcement(Version previousVersion) {
```

Comment:
I'm reading this wrong I think. If we've accidentally turned it on for everyone and this disable bit looks like it is only running if its upgrading from 1.82. Guess I just want to be sure that since we turned it "on" for everyone that we are turning it "off" -- I feel like I'm missing part of the puzzle here.

Reply:
or our get settings is turning it off in that call, and then we flip it on during install. I might be catching on now :)

Reply:
"If we've accidentally turned it on for everyone and this disable bit looks like it is only running if its upgrading from 1.82." -- we never accidentally messed up the "Disabled Preferred Phone Enforcement" setting. This worked as expected when existing customers were upgraded to 1.82 so this is fine as-is